docs/getting-started/08-connect-external-application.md

```diff
@@ -0,0 +1,278 @@
+5. Select **Register All** on the **Local APIs** tab or just register **SAP Commerce Cloud - Events** to be able to send events.
+
+Once registered, you will see all Commerce mock APIs and events available under the **Remote APIs** tab.
+
+> **TIP:** Local APIs are the ones available within the mock application. Remote APIs represent the ones registered in Kyma.
+
+### Expose events in the Namespace
+
+To expose events in the `orders-service` Namespace, first create an ApplicationMapping CR to bind the application to the Namespace. Then, provision the Events API in the Namespace using the ServiceInstance CR.
+
+Follow these steps:
+
+<div tabs name="expose-events-in-namespace" group="connect-external-application">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create an [ApplicationMapping CR](/components/application-connector/#custom-resource-application-mapping) and apply it to the cluster:
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+  apiVersion: applicationconnector.kyma-project.io/v1alpha1
+  kind: ApplicationMapping
+  metadata:
+    name: commerce-mock
+    namespace: orders-service
+  EOF
+  ```
+
+2. List available ServiceClass CR in the `orders-service` Namespace. Under the `EXTERNAL-NAME` column, find one with the `sap-commerce-cloud-events-*` prefix.
+
+   ```bash
+   kubectl get serviceclasses -n orders-service
+   ```
+   Copy the full `EXTERNAL NAME` to an environment variable. See the example:
+
+   ```bash
+   export EVENTS_EXTERNAL_NAME="sap-commerce-cloud-events-58d21"
+   ```
+
+3. Provision the Events API in the `orders-service` Namespace by creating a ServiceInstance CR:
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+  apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ServiceInstance
+  metadata:
+    name: commerce-mock-events
+    namespace: orders-service
+  spec:
+    serviceClassExternalName: $EVENTS_EXTERNAL_NAME
+    servicePlanExternalName: default
+  EOF
+  ```
+
+4. Check that the ServiceInstance CR was created. The last condition in the CR status should state `Ready True`:
+
+   ```bash
+   kubectl get serviceinstance commerce-mock-events -n orders-service -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+   ```
+
+  </details>
+  <details>
+  <summary label="console-ui">
+  Console UI
+  </summary>
+
+1. Back in the Console UI, go to **Integration** > **Applications/Systems** > **commerce-mock**.
+
+2. Select **Create Binding** to bind the application to the Namespace in which you will later provision the APIs provided by Commerce mock.
+
+3. Select the `orders-service` Namespace and click **Create**.
+
+4. Go to the `orders-service` Namespace view and navigate to **Service Management** > **Catalog**. Once on the **Services** tab, find **SAP Commerce Cloud - Events** and select it.
+
+   > **TIP:** You can also use the search in the upper right corner.
```

Comment:
```suggestion
   > **TIP:** You can also use the search bar in the upper right corner.
```
Or `search box`/`search engine`/`search mechanism`/`search tool`, not just `search` alone.
`Search` as a noun means an action or an attempt to look for something.